File: src/main/java/no/nav/domain/pensjon/regler/gomregning/PersongrunnlagOmregningResultat.java
package no.nav.domain.pensjon.regler.gomregning;

import no.nav.domain.pensjon.regler.grunnlag.Inntektsgrunnlag;

import java.io.Serializable;
import java.util.ArrayList;

public class PersongrunnlagOmregningResultat implements Serializable {

    private Long persongrunnlagId;
    private ArrayList<Inntektsgrunnlag> inntektsgrunnlagResultatListe;


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/gomregning/PersongrunnlagOmregningResultat.kt
package no.nav.pensjon.regler.domain.gomregning

import no.nav.pensjon.regler.domain.grunnlag.Inntektsgrunnlag
import java.io.Serializable

class PersongrunnlagOmregningResultat : Serializable {
    var persongrunnlagId: Long? = null
    var inntektsgrunnlagResultatListe: MutableList<Inntektsgrunnlag> = mutableListOf()

}